packages/ui-extensions/docs/surfaces/checkout/reference/targets/customer-account.order-status.block.render.doc.ts

```diff
@@ -11,7 +11,7 @@ const data: ReferenceEntityTemplateSchema = {
   name: 'customer-account.order-status.block.render',
   description: `A [block extension target](/docs/api/checkout-ui-extensions/extension-targets-overview#block-extension-targets) that renders exclusively on the **Order status** page. Unlike static extension targets, block extension targets render where the merchant sets them using the [checkout editor](/docs/apps/checkout/test-ui-extensions#test-the-extension-in-the-checkout-editor).
 
-  The [placements](/docs/api/checkout-ui-extensions/extension-targets-overview#placements) for block extension targets can be previewed during development by [using a URL parameter](/docs/apps/checkout/best-practices/testing-ui-extensions#block-extension-targets).
+  The [supported locations](/docs/api/checkout-ui-extensions/extension-targets-overview#supported-locations) for block extension targets can be previewed during development by [using a URL parameter](/docs/apps/checkout/best-practices/testing-ui-extensions#block-extension-targets).
```

Comment:
Seems linked to the change to `packages/ui-extensions-react/src/surfaces/checkout/render.tsx` above, can we keep it @jamesvidler ? 🙏  and same question for 2 files below (`packages/ui-extensions/docs/surfaces/checkout/reference/targets/purchase.checkout.block.render.doc.ts` and `packages/ui-extensions/docs/surfaces/checkout/reference/targets/purchase.thank-you.block.render.doc.ts`)